cypher/cypher-docs/src/docs/dev/ql/call/index.asciidoc

```diff
@@ -7,13 +7,13 @@ The `CALL` clause is used to call a procedure deployed in the database.
 --
 
 * <<query-call-introduction, Introduction>>
-* <<call-call-a-procedure, Call a procedure using `CALL`>>
+* <<call-call-a-procedure-using-call, Call a procedure using `CALL`>>
 * <<call-call-a-procedure-using-a-quoted-namespace-and-name, Call a procedure using a quoted namespace and name>>
 * <<call-call-a-procedure-with-literal-arguments, Call a procedure with literal arguments>>
 * <<call-call-a-procedure-with-parameter-arguments, Call a procedure with parameter arguments>>
 * <<call-call-a-procedure-with-mixed-literal-and-parameter-arguments, Call a procedure with mixed literal and parameter arguments>>
 * <<call-call-a-procedure-with-literal-and-default-arguments, Call a procedure with literal and default arguments>>
-* <<call-call-a-procedure-within-a-complex-query, Call a procedure within a complex query using `CALL...YIELD`>>
+* <<call-call-a-procedure-within-a-complex-query-using-call-yield, Call a procedure within a complex query using `CALL...YIELD`>>
```

Comment:
`CALL ...YIELD` is correct. `CallTest` has problems when including ellipses, so removed them in that context. `CallTest` will shortly be migrated onto the new framework, when this problem will cease to exist.